builder/steps/gen-dockerfile/contents/src/data/install-node-version

```diff
@@ -0,0 +1,4 @@
+# Check to see if the the version included in the base runtime satisfies
+# %s, if not then do an npm install of the latest available
```

Comment:
Thats a good point.  It's a typo in the docs.  I will need to double check exactly what the install node script does, but it does use an install method that does not use npm.  I will update the docs.